server/db/models/payment.js

```diff
@@ -0,0 +1,23 @@
+const { UUID, UUIDV4, FLOAT, STRING, ENUM } = require('sequelize');
+const db = require('../db');
+
+const Payment = db.define('payment', {
+  id: {
+    type: UUID,
+    defaultValue: UUIDV4,
+    primaryKey: true,
+  },
+  amount: {
+    type: FLOAT,
```

Comment:
no real affect on the mvp, but we should switch our amount types to DECIMAL(19, 4), this seems to be the standard for handling money in sql